src/d3a/blockchain/__init__.py

```diff
@@ -28,7 +28,7 @@
     from substrateinterface import SubstrateInterface
 
 
-DEFAULT_SUBSTRATE_URL = "ws://127.0.0.1:9944"
+DEFAULT_SUBSTRATE_URL = "wss://canvas-rpc.parity.io"
```

Comment:
Shall we control this also via an environmental variable, or is this an overkill for now?

Reply:
Probably an overkill because the canvas node is a temporary solution - this is a testnet running specifically for testing smart contracts